playbooks/acs.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: all
+- hosts: all:!external
```

Comment:
as a safety measure I would actually exclude explicitly all members of `external` from each and every role

Reply:
don't see how it could happen if hosts are referenced by group name in the playbook

Reply:
Indeed not you're right. I got lost in my reasoning here